### PR TITLE
Update docs regarding Ginkgo and Kokkos

### PR DESCRIPTION
**Pull request by davidscn**
according to precice/precice#2051

### CODE REVIEW

File: pages/docs/configuration/configuration-mapping.md [hunk 1]
```diff
@@ -131,11 +131,7 @@ ASTE and our [ASTE tutorial](tutorials-aste-turbine.html) enable full insight in
 
 #### Execution backends
 
-{% experimental %}
-Although this feature is well-tested in preCICE, it relies on an unreleased version of Ginkgo. Hence, this feature might change in the future.
-{% endexperimental %}
-
-Starting from version 3, preCICE offers to execute `mapping:rbf-global...` on different executor backends using the linear-operator library Ginkgo
+Starting from version 3.2, preCICE offers to execute `mapping:rbf-global...` on different executor backends using the linear-operator library Ginkgo in conjunction with Kokkos (build preCICE using `PRECICE_FEATURE_GINKGO_MAPPING=ON`)
```
Comment (fsimonis) [inline pages/docs/configuration/configuration-mapping.md]: No reason to keep this as a note.

```suggestion
Starting from version 3.2, preCICE offers to execute `mapping:rbf-global...` on different executor backends using the linear-operator library Ginkgo in conjunction with Kokkos.

To use this feature, please build preCICE from source with  [Ginkgo Mappings enabled](installation-source-dependencies.html#ginkgo).
```

File: pages/docs/installation/installation-source-dependencies.md [hunk 2]
```diff
@@ -257,6 +258,16 @@ Keep in mind that already [PETSc](installation-source-dependencies.html#petsc) s
...
+### Ginkgo
+
+| preCICE             | 3.2.0 >= |
+| Required Ginkgo     | 1.8.0 >= |
+| Required Kokkos     | 4.1.0 >= |
+
+[Ginkgo](https://ginkgo-project.github.io/) enables support for GPU- and OpenMP-accelerated (global) radial-basis function mappings. To enable the feature in preCICE, use the CMake option `-DPRECICE_FEATURE_GINKGO_MAPPING=ON`. Using this feature in preCICE requires additionally [Kokkos](https://kokkos.org/).
+
+Both packages follow the usual CMake build instructions and can be downloaded on the GitHub release page for [Ginkgo](https://github.com/ginkgo-project/ginkgo/releases) and [Kokkos](https://github.com/kokkos/kokkos/releases). For Kokkos, system packages might be available through your package manager as well.
+
```
Comment (fsimonis) [inline pages/docs/installation/installation-source-dependencies.md:270]: Maybe a useful hint would be how to deactivate building tests and examples for Ginkgo, which takes a while.

**Review by fsimonis — APPROVED** [review]
Fine after minor extensions

File: pages/docs/installation/installation-source-dependencies.md (hunk 2 above)
Comment (davidscn) [reply pages/docs/installation/installation-source-dependencies.md:270]: yes

**Pull request merged**